src/controller/python/chip/yaml/parser.py

```diff
@@ -52,13 +211,14 @@ def run_action(self, dev_ctrl: ChipDeviceCtrl, endpoint: int, node_id: int):
 class InvokeAction(BaseAction):
     '''Single invoke action to be executed including validation of response.'''
 
-    def __init__(self, item: dict, cluster: str, data_model_lookup: DataModelLookup):
+    def __init__(self, item: dict, cluster: str, reference_objects: _ReferenceObjects):
         '''Parse cluster invoke from yaml test configuration.
 
         Args:
           'item': Dictionary containing single invoke to be parsed.
           'cluster': Name of cluster which to invoke action is targeting.
-          'data_model_lookup': Data model lookup to get attribute object.
+          'reference_objects': Contains global common objects such has data model lookup, storage
```

Comment:
```suggestion
          'reference_objects': Contains global common objects such as data model lookup, storage
```

What does "data model lookup" mean?

Reply:
When looking at the code, "ReferenceObjects" is more "ScriptExecutionContext"